INSTALLATION.md

```diff
@@ -1,76 +1,76 @@
+
 # Installation
 
 * [Introduction](#introduction)
-* [Step by step instructions](#step-by-step-instructions)
+* [Desktop](#desktop)
   * [MacOS](#macos)
   * [Linux](#linux)
   * [Windows](#windows)
 
 ## Introduction
 
-This document will guide you through the installation process of the latest tagged version of Thyra.
-
-> **_PREREQUISITES:_** Be comfortable with your terminal and have a recent version of MacOS, Windows or Linux.
-
-> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
+From this point you'll start to custom your Massa's journey thanks to Thyra. If you want to learn more about Thyra's objectives and mindset, go here; if you're curious about the technical aspects and want to contribute go [there](./CONTRIBUTING.md).
```

Comment:
it looks like a link is missing here: go here?

Reply:
We discussed about it [here](https://github.com/massalabs/thyra/pull/518#discussion_r1152222534)